<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DuplicateOpOptionsWidget</class>
 <widget class="QWidget" name="DuplicateOpOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>87</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbHealing">
     <property name="text">
      <string>Healing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbPerspective">
     <property name="text">
      <string>Correct the perspective</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbSourcePoint">
     <property name="text">
      <string>Source point move</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
